tests/unit/utils/parsing_test.py

```diff
@@ -116,17 +117,20 @@
 EXPECTED_METADATA = {
     'weekday': 'Wed',
     'month': 'Mar',
-    'day': '8',
+    'day': 8,
     'time': '09:25:20',
-    'year': '2023',
+    'year': 2023,
     'version_1': 'EYELINK II 1',
     'version_2': 'EYELINK II CL v6.12 Feb  1 2018 (EyeLink Portable Duo)',
     'model': 'EyeLink Portable Duo',
-    'version_number': '6.12',
-    'sampling_rate': '1000.00',
+    'version_number': 6.12,
+    'sampling_rate': 1000.00,
     'filter': '2',
     'tracking': 'CR',
     'tracked_eye': 'LEFT',
+    'calibrations': [],
+    'validations': [],
+    'resolution': (0, 0, 1279, 1023),
```

Comment:
I would expect the resolution tuple to be `(1280, 1024)`.

This 4-tuple here rather refers to the coordinates of two opposing screen edges.

As the pixel counting starts with zero in the EyeLink system, the actual resolution should be calculated by `(coords[3] - coords[1] + 1, coords[4] - coords[2] + 1)`